core/src/Cuda/Kokkos_Cuda_UniqueToken.hpp

```diff
@@ -60,14 +59,19 @@ namespace Experimental {
 template <>
 class UniqueToken<Cuda, UniqueTokenScope::Global> {
  protected:
-  uint32_t volatile* m_buffer;
-  uint32_t m_count;
+  Kokkos::View<uint32_t*, Kokkos::CudaSpace> m_locks;
 
  public:
   using execution_space = Cuda;
   using size_type       = int32_t;
 
-  explicit UniqueToken(execution_space const& = execution_space());
+  explicit UniqueToken()
+      : m_locks(Kokkos::View<uint32_t*, Kokkos::CudaSpace>(
+            "Kokkos::UniqueToken::m_locks", Kokkos::Cuda().concurrency())) {}
```

Comment:
```suggestion
      : m_locks("Kokkos::UniqueToken::m_locks", Kokkos::Cuda().concurrency()) {}
```
haven't checked the format